src/main/java/com/avairebot/audio/GuildMusicManager.java

```diff
@@ -48,8 +48,8 @@
 
     private boolean hasSetVolume;
     private boolean hasPlayedSongBefore;
-    private boolean repeatQueue;
     private int defaultVolume;
+    private int repeatQueue = 0;
```

Comment:
It might be a good idea to use an Enum for the repeat queue state instead of an integer, it would make the code less ambiguous a lot more readable.